cmd/admxgen/defs/mount.yaml

```diff
@@ -0,0 +1,11 @@
+- key: "/mount"
+  displayname: "Mount points for the client"
+  explaintext: |
+    Define mount points that will be mounted for the client.
+    The order of the mounts will be according to the order the mount points were listed in the policy.
+    The mounts listed will be appended to the list of mounts referenced higher in the GPO hierarchy.
```

Comment:
Precise that the mounts will be deduplicated. Also that both anynomous mount and those needing kerberos credentials. are handled.

And also, maybe precise the format of the text mount, like protocol:///?
You can add some example like, the anonymous tag and one with a protocol, telling it’s going to try to be authenticated in comments.